examples/courses/2018-11-SenSys/sensys_udp_rx/main.c

```diff
@@ -101,6 +101,7 @@ int main(void) {
       break;
   }
 
-  /* Tock keeps the app alive waiting for callbacks after
-   * returning from main, so no need to busy wait */
+  while (1) {
```

Comment:
```suggestion
  // Note: Prior to 2.0, libtock-c kept apps alive in an implicit busy loop.
  // Since 2.0, apps must now explicitly include their own event loop to
  // avoid termination.
  while (1) {
```

Give some indicator that something changed in case there's supporting text out somewhere that points to this.